src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/Telemetry/AggregatingTelemetryLogManager.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using Microsoft.VisualStudio.Telemetry;
+
+namespace Microsoft.VisualStudio.Razor.Telemetry;
+
+/// <summary>
+/// Manages creation and obtaining aggregated telemetry logs.
+/// </summary>
+internal sealed class AggregatingTelemetryLogManager : IDisposable
```

Comment:
Taken from https://github.com/dotnet/roslyn/blob/main/src/VisualStudio/Core/Def/Telemetry/Shared/AggregatingTelemetryLogManager.cs

Reply:
@dibarbet has a PR out removing the roslyn version of this class. You might want to coordinate with him before duplicating it.

[https://github.com/dotnet/roslyn/pull/75402](https://github.com/dotnet/roslyn/pull/75402)